shared/folders/dumb.js

```diff
@@ -315,25 +315,26 @@ const commonFiles = (isPrivate): FilesProps => ({ // eslint-disable-line arrow-p
     {name: 'Yesterday', modifiedMarker: false, files: genFiles(4, 4, isPrivate)},
   ],
   recentFilesEnabled: true,
+  onClickPaperkey: device => console.log('on click paperkey ', device),
 })
 
 const commonParticipant = {
   recentFilesSection: [],
   waitingForParticipantUnlock: [
-    {name: 'throughnothing', devices: 'Tell them to turn on: Home Computer, ben\'s iPhone or Work laptop.', onClick: () => console.log('clicked throughnothing')},
-    {name: 'bob', devices: 'Tell them to turn on bob\'s Android phone', onClick: () => console.log('clicked bob')},
+    {name: 'throughnothing', devices: 'Tell them to turn on: Home Computer, ben\'s iPhone or Work laptop.'},
```

Comment:
🐑 🐑 🐑 

The formatting in code uses oxford comma, so add a comma before `or` to match it